ReSharper.FSharp/test/src/FSharp.Tests/Parsing/ArgumentsOwnerTest.fs

```diff
@@ -0,0 +1,170 @@
+﻿namespace JetBrains.ReSharper.Plugins.FSharp.Tests.Features
+
+open System
+open FSharp.Compiler.SourceCodeServices
+open JetBrains.ProjectModel
+open JetBrains.ReSharper.Feature.Services.ExpressionSelection
+open JetBrains.ReSharper.FeaturesTestFramework.Utils
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Impl.DeclaredElement
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Tree
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Util
+open JetBrains.ReSharper.Plugins.FSharp.Tests.Common
+open JetBrains.ReSharper.Plugins.FSharp.Util
+open JetBrains.ReSharper.Psi.Tree
+open JetBrains.ReSharper.TestFramework
+open JetBrains.TextControl
+open NUnit.Framework
+
+[<AttributeUsage(AttributeTargets.Method, AllowMultiple=false)>]
+type ExpectErrorsAttribute () =
+    inherit Attribute()
+
+[<FSharpTest>]
+[<TestPackages("FSharp.Core")>]
+[<TestReferences("System.Drawing", "System", "System.Core")>]
+[<TestProjectOutputType(ProjectOutputType.CONSOLE_EXE)>]
```

Comment:
I think it'd be great to add `ITestProjectOutputTypeProviderAttribute` to the framework and to make `FSharpTestAttribute` implement it, defaulting to `ProjectOutputType.CONSOLE_EXE`. It'll allow override it on per-test basis but having `Exe` default will save us from having to add `module Module` everywhere. I'll look into it.

Reply:
I'll do that when I split the test infra changes to be their own PR 👍